spec/presenters/two_factor_authentication/set_up_phone_selection_presenter_spec.rb

```diff
@@ -1,10 +1,10 @@
 require 'rails_helper'
 
-RSpec.describe TwoFactorAuthentication::PhoneSelectionPresenter do
+RSpec.describe TwoFactorAuthentication::SetUpPhoneSelectionPresenter do
   let(:user_without_mfa) { create(:user) }
   let(:user_with_mfa) { create(:user, :with_phone) }
-  let(:presenter_with_mfa) { described_class.new(configuration: phone, user: user_with_mfa) }
-  let(:presenter_without_mfa) { described_class.new(configuration: phone, user: user_without_mfa) }
```

Comment:
These appear to have been the only references to the `phone` variable. Can we remove the assignment of it on lines 11, 31, and 45?